jctools-experimental/src/main/java/org/jctools/channels/proxy/ProxyChannelFactory.java

```diff
@@ -342,7 +355,20 @@ private static void implementProcess(ClassVisitor classVisitor,
         // return i;
         methodVisitor.visitVarInsn(Opcodes.ILOAD, localIndexOfLoopIndex);
         methodVisitor.visitInsn(Opcodes.IRETURN);
+        
 
+        // Label the end of scope for all locals
+        Label localScopeEnd = new Label();
+        methodVisitor.visitLabel(localScopeEnd);
+
+        // Declare local variables to aid in debugging
```

Comment:
Self explanatory. This names the locals in the debugger variables page when looking at this generated frame.